pyvista/plotting/plotting.py

```diff
@@ -278,6 +278,22 @@ def __init__(
         if self.theme.antialiasing:
             self.enable_anti_aliasing()
 
+        # widgets
+        self.camera_widgets = []
+        self.box_widgets = []
+        self.box_clipped_meshes = []
```

Comment:
And a concern that might be for another PR: is it on purpose that `self.plane_sliced_meshes` etc. mesh lists are never cleared, only appended to? When we disable a widget, do we want to keep these meshes around? Or should we clear the mesh lists as well?

Reply:
I think it's ok to keep the plane sliced meshes since we're only disabling the widget, but we probably should make not of it in the docstring.